app/views/bulkrax/shared/_entries_tab.html.erb

```diff
@@ -12,5 +12,5 @@
       </tr>
     </thead>
   </table>
-  <div id='importer-entry-classes' class='hidden'><%= [@importer.parser.entry_class.to_s, @importer.parser.collection_entry_class.to_s, @importer.parser.file_set_entry_class.to_s].compact.join('|') %></div>
+  <div id='<%= "importer-entry-classes" %>' class='hidden'><%= [item.parser.entry_class.to_s, item.parser.collection_entry_class.to_s, item.parser.file_set_entry_class.to_s].compact.join('|') %></div>
```

Comment:
'importer-entry-classes' is just a label for the datatables JS to key off of. it doesn't need to be erb and can be renamed importexport-entry-classes or something